app/assets/javascripts/components/createUser.jsx

```diff
@@ -217,10 +216,11 @@ class CreateUser extends React.Component {
                 <i className="fa fa-success"></i>
                  <span>{this.state.successMessage}</span>
                 </div> : null }
-              { this.state.showFailed ? <div className="error-info" >
-                  <i className="fa fa-error"></i>
+                <div className={this.state.showFailed ? 'error-info' : ''} >{ this.state.serverErrors.length && this.state.showFailed ? this.state.serverErrors.map((err, i) => {
```

Comment:
For readability reasons, I will do 
{this.state.showFailed ? <div className="error-info" > {this.serverErrors.length? this.state.serverErrors.map((err, i) => { return <p className="error center-align" key={i}> {err} </p> }): <span>{this.state.errorMessage}</span></div> : null }